UOP1_Project/Assets/Scripts/StateMachine/Editor/Utilities/TransitionDisplayHelper.cs

```diff
@@ -46,7 +46,14 @@ internal bool Display(ref Rect position)
 				LabelField(rect, "To");
 
 				rect.x += 20;
-				LabelField(rect, SerializedTransition.ToState.objectReferenceValue.name, EditorStyles.boldLabel);
+				if (SerializedTransition.ToState.objectReferenceValue != null)
+				{
+					LabelField(rect, SerializedTransition.ToState.objectReferenceValue.name, EditorStyles.boldLabel);
+				}
+				else
+				{
+						LabelField(rect, "Unknown", EditorStyles.boldLabel);
+				}
```

Comment:
Actually, I am checking and logging it in `TransitionTableEditor.cs`, but there's an error in the script!
``` csharp
if (serializedTransition.FromState.objectReferenceValue == null)
{
	Debug.LogError("Transition with invalid state found in table " + serializedObject.targetObject.name + ", deleting...");
	_transitions.DeleteArrayElementAtIndex(i);
	serializedObject.ApplyModifiedProperties();
	Reset();
	return;
}
if (serializedTransition.FromState.objectReferenceValue == null)
{
	Debug.LogError("Transition with invalid state found in table " + serializedObject.targetObject.name + ", deleting...");
	_transitions.DeleteArrayElementAtIndex(i);
	serializedObject.ApplyModifiedProperties();
	Reset();
	return;
}
```
I'm checking `FromState` twice!! 🤦🏻

Reply:
My bad, indeed, I should not have added this null check in the middle of this PR... 
Indeed, I did that, because I experienced fatal exception in case of state deletion: Transition Table was crashing when opening it if a ToState was missing! 

I better understand why with your last comment! It is indeed better like that :) I will update my PR to remove the null check!

BTW: I did not follow actively the SM thread (I am already bothered on other topics and I am mostly using my free time to contribute), despite that it was easy to understand how it works and how to use it to build the behavior I wanted to put in place, especially without having to write lots of boiler plate code! Great job! 
BTW2: I have also learned lots of things reading the code you have written here!